config/environments/development.rb

```diff
@@ -26,7 +26,7 @@
 
   config.action_mailer.smtp_settings = {
     address: '127.0.0.1',
-    port: 25,
+    port: 1025,
```

Comment:
Hmm, not sure about this one.
We had errors on dev.fsektionen.se and then I couldn't use mailer locally either, so I changed the port.
What do you think?
@fsek/admins